### ISSUE TITLE
Forbidden when trying to login to github

### ISSUE THREAD

**Issue by SamHep0803** [cdr/code-server#2111]
iPad Test
- Web Browser: Safari
- Local OS: iPadOS 14.0
- Remote OS: Ubuntu 20.04
- Remote Architecture: x64
- `code-server --version`: `3.5.0 de41646fc402b968ca6d555fdf2da7de9554d28a`

Windows Machine
- Web Browser: Brave Browser (basically just chromium)
- Local OS: Windows 10 2004 x64
- Remote OS: Ubuntu 20.04 (same machine as above)
- Remote Architecture: x64
- `code-server --version`: `3.5.0 de41646fc402b968ca6d555fdf2da7de9554d28a`

When I try to clone a private GitHub repository and `code-server` asks me to open the GitHub authentication website, it opens, I press on continue then I just get a "Forbidden" page.

![image](https://user-images.githubusercontent.com/14109339/93599360-112e1280-f9cf-11ea-93f5-cd370ef68262.png)

Please advise on what to do here as I would like to get this repo on this machine so that I can develop.

**Comment by SamHep0803** [comment]
For now I have turned of auto github authentication but I would like to see if auto works.

**Comment by code-asher** [comment]
Yeah, we need to figure out how to make GitHub authentication work. :thinking:  Tracking here: https://github.com/cdr/code-server/issues/1883

**Comment by OliverwengFiltered** [comment]
use ssh. and set deployment key to your github repo.

this is what I use.
